less/components/map.less

```diff
@@ -20,3 +20,25 @@
   position: relative;
   height: 100%;
 }
+
+.marker-popup {
+  padding-left: 0;
+  margin-bottom: 0;
+  list-style-type: none;
+  li {
+    &:first-child {
+      border-top: none;
```

Comment:
Oh yea, I noticed this as well.  I think that day I jumped to another branch and just committed whatever code changes I made. I knew some more tweaks would be made! . ...so yea smile_cat: